### DISCUSSION TITLE
Audio Input Device for Web

### DISCUSSION THREAD

**Discussion by bypie5** [godotengine/godot-proposals#10260] [Platforms]
It looks like selecting an audio input device, such as a microphone, is not [currently not supported for the web platform](https://docs.godotengine.org/en/stable/tutorials/audio/recording_with_microphone.html). Although adding a microphone to a game is currently a somewhat niche feature for web, I think that games that use proximity/voice chat or see users interacting with an LLM might benefit from support for this.

Has there been any investigation for adding support for audio input devices on web?

**Comment by goatchurchprime** [comment]
Web microphone input seems to be fine on Godot 4.3 here: https://goatchurch.itch.io/twovoip-mqtt
